liveRecorder/Tasks/ConcatinationTask.py

```diff
@@ -126,6 +126,7 @@ def get_flavor_id(self, url_postfix, single_flavor):
 
     def run(self):
 
+        self.reset_retry_count(self.recording_path)
```

Comment:
both ConcatinationTask.py and UploadTask.py uses it, please move it to the base class

Reply:
moved reset_retry_count to BaseTask.py